<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XNStealer Builder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Webhook and FTP Settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Discord Webhook</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="WebHook_Text">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FTP Server</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ftpServer_Text">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FTP Name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ftpName_Text">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FTP Pass</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ftpPass_Text">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="CheckBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Check FTP</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="AboutBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BuildBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Build Loader</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Advanced Settings</string>
    </property>
    <widget class="QCheckBox" name="AntiVM_Box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Anti-VM</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BypassUAC_Box">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bypass-UAC</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BypassFW_Box">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bypass-FireWall</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="stealFiles_Text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="plainText">
      <string>&quot;.log&quot;,&quot;.db&quot;,&quot;.bakdb&quot;,&quot;.csv&quot;,&quot;.sqlite&quot;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Steal Files</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
